app/Http/Controllers/HomeController.php

```diff
@@ -2,17 +2,24 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Models\Course;
+use App\Models\UserCourse;
+use App\Models\Lesson;
+use App\Models\Review;
 
 class HomeController extends Controller
 {
     /**
      * Show the application dashboard.
      *
-     * @return \Illuminate\Contracts\Support\Renderable
      */
     public function index()
     {
-        return view('home');
+        $courses = Course::main()->get();
+        $reviews = Review::main()->get();
+        $totalCourse = Course::count();
+        $totalLesson = Lesson::count();
+        $learners = UserCourse::learner()->get()->count();
```

Comment:
không cần get mà count luôn được không?

Reply:
phải có get anh ạ. Tại bảng đấy là bảng nối 2 bảng nên phải get để lấy dữ liệu 2 bảng kia ra xong mới đếm đc

Reply:
@Nguyen-Bao-Anh đã thử trong code chưa?

Reply:
@hapo-nghialuu e thử r ạ, nếu ko có get thì nó chỉ đếm được 1 thôi còn có get thì nó mới đếm được toàn bộ